report.md

```diff
@@ -1,27 +1,28 @@
-# 报告问题的正确姿势
+# How to request question
 
-任何事物都不可能尽善尽美，LittleSkin 也是如此。
+Everything are not possible to perfect, LittleSkin is also like this.
 
-如果你在使用 LittleSkin 的过程中遇到了任何问题（包括但不限于材质无法显示、网站出现 Bug 等），不要气急败坏，请充分理解下述内容后正确提问。
+If you have any problem that you use LittleSkin (including but not limited to: texture can't display, website have bugs etc.), don't utterly discomfited， please understand the following content fully, and then ask question correctly.
 
-## 遇到问题后的第一反应
+## The first reaction when having a problem
 
 > DON'T PANIC.
 
-首先，你应该先检查一下是不是你的问题。大部分情况下，LittleSkin 是可以正常运行的。如果你遇到了错误，请先确定这是不是你自己的锅：网络错误、操作错误、甚至是你自己的浏览器出问题都可能导致你无法正常使用 LittleSkin。所以，在提问之前，请先以自己的经验判断一下，这是不是你的问题。
+First, you should check if this is your problem at first. At many times, LittleSkin can run normally. If you get an error, please make sure that this isn't your problem at first: Network error, Operating error, even your browser error might can't let you use LittleSkin normally. So, before asking questions, please judgment with your experience, this is your problem or not.
 
-其次，你应该先阅读一遍 [常见问题解答](/faq.html)。很多情况下你遇到的问题也曾经困扰过很多人，并且已经被总结出了解决方案。这种情况下，直接查阅 FAQ 能更快地解决你的问题。
+Second, you should read [FAQ](/faq.html) at first. At many times, your problem might also trouble many people, and these problem are already have solution. At this situation, read FAQ directly can solve your problem fastly.
 
-如果你确定你没有问题，并且你遇到的问题不在 FAQ 中，你可以着手准备报告问题了。
+If you sure that you don't have any problem, and your problem isn't in FAQ, you can prepare to report problem.
 
-:::tip 推荐阅读：<a href="https://github.com/ryanhanwu/How-To-Ask-Questions-The-Smart-Way/blob/master/README-zh_CN.md" target="_blank">《提问的智慧》</a>
+:::tip We recommend you to read：<a href="http://www.catb.org/~esr/faqs/smart-questions.html" target="_blank">How To Ask Questions The Smart Way</a>
 
-充分理解这篇文章可以帮助你更好地提问并更快地收到回复和正确的解答（不论何时何地）。但请注意，《提问的智慧》的创作团队仅是提供这一篇文章来指导你如何正确提问，他们并不提供针对任何问题的任何技术支持。**不要做白痴。**
+Understang this article fully can help you ask questions better, and also receive the reply and the right answer fastly. But please attention, the team of How To Ask Questions The Smart Way is only provide this article to guide you how to ask questions correctly, they don't provide any technical support of any problem. **Don't be an idiot.**
```

Comment:
> Understang this article fully

Fully understand this article

> fastly

**quickly** !!!

> But please attention

But please **pay** attention